web-docs/docs/_partial/_sys_req_hardware.mdx

```diff
@@ -0,0 +1,37 @@
+import ScDocusaurusConfig from "@site/ScDocusaurusConfig.js";
+
+<>The {ScDocusaurusConfig.title} has been designed to take advantage of the Ignition server's available hardware
+resources. Multi-threading and asynchronous processing allow the connector to simultaneously handle multiple requests at
+a time. </><br/>
+
+#### Minimum Hardware Requirements
+
+CPU: 2 cores<br/>
+RAM: 4 GB<br/>
+
+:::caution
+
+The minimum hardware requirements are suitable for a small number of devices and tags. If you are planning to connect
+many devices or tags, you should consider increasing the hardware resources available to Ignition and its modules.
+
+These requirements match those of Ignition, with additional memory (RAM) required for the connector to run.
```

Comment:
How was this determined

Reply:
There is no advanced determination here. It is simply the Ignition requirements with additional memory to support the connector. The connector will use at least some memory, but the amount of tags/Ewons really controls the amount. 

The idea of adding a minimum beyond what the Ignition defaults are is that 'guarantees' that the connector has its own dedicated resources of some amount. It also brings to attention that system resources, including memory, must be considered, as mentioned in the larger comment for the recommended hardware.

In many cases, the minimum hardware requirements may be adequate (for example, my developer/testing account with only a handful of Ewons + date has no issue with 4 GB). The minimum requirement of 4 GB can still cause concern on large customer systems, though, where hundreds of tags and Ewons are in place. Upping to 8 GB, in that case, may improve the connector performance and reduce the change of Ignition time/clock drift due to the connector overloading the system.